exercises/concept/lasagna/.docs/hints.md

```diff
@@ -4,10 +4,20 @@
 
 - An [integer value][integers] can be defined as one or more consecutive digits.
 
+If you see a `panic:` error when running the tests, this is as you haven't implemented one of the functions (it should say which one) or you have left the boilerplate in place, you need to remove the `panic(...)` line from the supplied code.
```

Comment:
```suggestion
If you see a `panic:` error when running the tests, this is because you have not implemented one of the functions (it should say which one) or you have left the boilerplate in place.
You need to remove the `panic(...)` line from the supplied code and replace it with a real implementation.
```